certbot-apache/tests/http_01_test.py

```diff
@@ -125,6 +125,18 @@ def test_configure_multiple_vhosts(self):
                 domain="duplicate.example.com", account_key=self.account_key)]
         self.common_perform_test(achalls, vhosts)
 
+    def test_configure_name_and_blank(self):
+        domain = "certbot.demo"
+        vhosts = [v for v in self.config.vhosts if v.name == domain or not v.get_names()]
```

Comment:
It'll probably never matter, but I figure we might as well update this logic too so it matches and if a vhost with only a serveralias is added to this Apache config, this test still works.
```suggestion
        vhosts = [v for v in self.config.vhosts if v.name == domain or v.name is None]
```
Otherwise, this LGTM!